src/Microsoft.Extensions.DependencyInjection.Ordered/OrderedServiceDescriptor.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.DependencyInjection.Ordered
+{
+    public class OrderedServiceDescriptor: EnumerableServiceDescriptor
```

Comment:
There is already IOrderedEnumerable but it has https://msdn.microsoft.com/en-us/library/bb548559(v=vs.110).aspx method on it.